pkg/city4cfd/metadata.json

```
@@ -0,0 +1,9 @@
+{
+    "name": "city4cfd",
+    "repo": "https://github.com/tudelft3d/City4CFD",
+    "description": "Reconstruction of 3D city geometries for urban CFD simulations",
+    "type": "app",
+    "build": "",
```

I think we can use an empty list of steps for build
```suggestion
    "build": [],
```